src/DankCord/DankCord.py

```diff
@@ -194,16 +197,20 @@ def run_sub_command(self, /, name: str, sub_name: str, retry=False, attempts=10,
         },
         "nonce": nonce
     }
-    message_object = False
     for i in range(attempts):
       requests.post("https://discord.com/api/v9/interactions", json.dumps(data), http_headers=[("Authorization", self.token), ("Content-Type", "application/json")])
       message_data = self._confirm_command_ran(nonce, timeout)
       status = not message_data is None
       if retry is False or status is True:
         break
     if status:
+      while not str(nonce) in self.ws_cache.keys():
+        time.sleep(0.00001)
```

Comment:
Why time.sleep instead of just continue?

Reply:
Changed